### DISCUSSION TITLE
Examples on Adobe's online Storybook run well but it not show SOURCE CODE!

### DISCUSSION THREAD

**Discussion by vinhpt2** [adobe/spectrum-web-components#4245] [General]
Examples on Adobe's online Storybook run well but it not show SOURCE CODE. Please share the source code of the examples. Thanks!

**Comment by Westbrook** [comment]
Code examples are available all throughout the main documentation site available at https://opensource.adobe.com/spectrum-web-components/ we do not currently have plans to expand code samples beyond that.
